packages/mobile/src/alert/reducer.ts

```diff
@@ -3,6 +3,7 @@ import { ErrorMessages } from 'src/app/ErrorMessages'
 import { RootState } from 'src/redux/reducers'
 
 export interface State {
+  errorType: 'show' | 'log'
```

Comment:
Adding a new `errorType` so that we can log errors for use with inline error messaging without having the error banner pop up

Reply:
I feel like it might be better to just call Logger directly. as if we are not updating UI why pay the cost of updating redux state?

Reply:
That's a good naming suggestion. Will do.

@aaronmgdr that's a good point and highlights my poor naming decision. We actually do want to `show` and not `log`, we just want it to show inline rather than in banner. I think Conner's suggestion clears up that confusion.